packages/mask/src/components/InjectedComponents/ProfileCard/ProfileCardTitle.tsx

```diff
@@ -65,9 +65,9 @@ export const ProfileCardTitle: FC<ProfileCardTitleProps> = ({
 
     const userId = identity.identifier?.userId
     const { value: nextIdBindings = EMPTY_LIST } = useAsync(async () => {
-        if (!userId) return EMPTY_LIST
-        return NextIDProof.queryProfilesByTwitterId(userId)
-    }, [userId])
+        if (!socialAccounts[0].label) return EMPTY_LIST
+        return NextIDProof.queryProfilesByDomain(socialAccounts[0].label)
+    }, [socialAccounts])
```

Comment:
Depend on critical value.
```suggestion
    }, [socialAccounts[0].label])
```